### DISCUSSION TITLE
Could not download "with-tailwindcss" because of a connectivity issue between your machine and GitHub

### DISCUSSION THREAD

**Discussion by gaterunner341** [tailwindlabs/tailwindcss#9239] [Help]
Attempting to npx a next app with tailwindcss. Receive the below errors: I believe this is the right place for this?

PS C:\Users\$USER\Desktop\cms_blog> npx create-next-app -e with-tailwindcss .
Creating a new Next.js app in C:\Users\$USER\Desktop\cms_blog.

Downloading files for example with-tailwindcss. This might take a moment.

? Could not download "with-tailwindcss" because of a connectivity issue between your machine and GitHub.
√ Could not download "with-tailwindcss" because of a connectivity issue between your machine and GitHub.
Do you want to use the default template instead? ... no

Aborting installation.
Unexpected error. Please report it as a bug:
 DownloadError: Premature close
    at createApp (C:\Users\$USER\AppData\Local\npm-cache\_npx\efeac22998af9bf2\node_modules\create-next-app\dist\index.js:317:17049)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (C:\Users\$USER\AppData\Local\npm-cache\_npx\efeac22998af9bf2\node_modules\create-next-app\dist\index.js:317:23085)

**Comment by wongjn** [comment]
With regards to these lines:
```
? Could not download "with-tailwindcss" because of a connectivity issue between your machine and GitHub.
√ Could not download "with-tailwindcss" because of a connectivity issue between your machine and GitHub.
```
Seems like an issue with your connectivity to GitHub, whether GitHub was down for a moment or some routing issue. Either way, this looks more like an issue with `create-next-app` rather than Tailwind, so I would ask over there!
